components/honoraryMember.tsx

```diff
@@ -70,7 +70,7 @@ const HonoraryMember = ({ member }: { member: MemberProps }) => {
         {!isMobile && displayRole}
         <p />
       </Stack>
-      <div className={styles.verticleLine} />
+      <hr className={styles.verticleLine} />
```

Comment:
Fix spelling?
verticleLine -> verticalLine

Reply:
thanks for noticing. I recall vercel not working properly at a previous occasion but after merging it didn't seem to have any effect. I'm not sure about what the problem could be but I'm open to suggestions.